app/src/main/res/navigation/nav_graph.xml

```diff
@@ -25,7 +25,9 @@
 
     <action
         android:id="@+id/action_global_search_dialog"
-        app:destination="@id/searchDialogFragment" />
+        app:destination="@id/searchDialogFragment"
```

Comment:
I tested this with the following STR:
1. Open a webpage
2. Open the tab trays
3. Go to Android Home
4. Click on the search widget

I still get the described actual behaviour in https://github.com/mozilla-mobile/fenix/issues/16807 where the search fragment isn't actually shown and the awesomebar isn't functional until I do a "back" to navigate back in the back stack.

Reply:
lol I totally misunderstood the original bug and was just fixing another apparently where you can get 2 search dialogs overlying eachother.